src/__tests__/rendering.test.js

```diff
@@ -0,0 +1,50 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from '../App';
+import store from '../redux/configureStore';
+import Navbar from '../components/Navbar';
+import Crypto from '../components/Crypto';
+import Home from '../components/Home';
+
```

Comment:
- [ ] As the previous reviewer mentioned there is an error thrown in the console, that's related to **circular dependency**, kindly fix this error so your app runs properly without errors

- Browser console:
![Screenshot_20221013_045354](https://user-images.githubusercontent.com/25227022/195495910-994986ee-7c53-4eb9-813f-4d0be0106dbe.png)
- Tests output:
![Screenshot_20221013_045926](https://user-images.githubusercontent.com/25227022/195496013-bf093f92-9c5b-4bd7-ac08-1b3da5d369c6.png)